internal/nop_tarball.go

```diff
@@ -12,8 +14,8 @@ type NOPTarBall struct {
 	tarWriter *tar.Writer
 }
 
-func (tarBall *NOPTarBall) SetUp(crypter Crypter, params ...string) {}
-func (tarBall *NOPTarBall) CloseTar() error                         { return nil }
+func (tarBall *NOPTarBall) SetUp(crypter crypto.Crypter, params ...string) {}
+func (tarBall *NOPTarBall) CloseTar() error                                { return nil }
```

Comment:
fix it

Reply:
I think there is nothing to fix. Indentation is changed because the previous line is changed.